src/main/scala/files_metadata/infrastructure/MetadataControllers.scala

```diff
@@ -622,4 +622,35 @@ class MetadataControllers {
       case e: Exception => _handleException( e )
     }
   }
+
+  def DeleteFileController(
+      request: cask.Request,
```

Comment:
It seems that you don't need the `request` object.  You can remove it from the controller to avoid IDE warnings.